cargo-pgrx/src/command/schema.rs

```diff
@@ -545,6 +568,12 @@ fn compute_sql(profile: &CargoProfile, package_name: &str) -> eyre::Result<()> {
     command.stdout(Stdio::inherit());
     command.stderr(Stdio::inherit());
 
+    // pass the package version through as an environment variable
+    command.env(
+        "PGRX_PKG_VERSION",
+        manifest.package_version().expect("`Cargo.toml` is missing the package version property"),
+    );
```

Comment:
I would prefer if this checked if `std::env::var("CARGO_PKG_VERSION")` is set and only when it is unset do the needful, and when we do that, **lie**: set Cargo's env vars for it, and only set the env vars we need. This avoids adding another quasi-stable quasi-unstable undocumented interface to our code.

Reply:
okay.